api/managementpb/backup/backups.proto

```diff
@@ -74,4 +192,32 @@ service Backups {
       body: "*"
     };
   }
+  // ScheduleBackup schedules repeated backup.
+  rpc ScheduleBackup(ScheduleBackupRequest) returns (ScheduleBackupResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/backup/Backups/Schedule"
```

Comment:
For consistency it probably should be `/v1/management/backup/Backups/AddScheduled`

Reply:
@atymchuk ^what do you think?

Reply:
I would go with either `Schedule` or `CreateScheduled`. Agree with @artemgavrilov, consistency is important, what's the equivalent endpoint for creating an on-deman/ad-hoc backup?

Reply:
As I can see it's the `Add` verb which is used everywhere. 
On consistency:
To me, consistency is a trap in the sense that once you land a sub-optimal term or verb, you have to stick to it forever. Which is wrong, because we want people to be able to read the code and understand it.

That said, IMO the best semantics here is "As a System, I'm scheduling a backup" and therefore David's version of "/v1/management/backup/Backups/Schedule" seems to be just OK.